cl/search/tasks.py

```diff
@@ -1098,9 +1098,14 @@ def index_parent_or_child_docs(
                 child_instances = RECAPDocument.objects.filter(
                     pk__in=instance_ids
                 )
-                parent_ids = RECAPDocument.objects.filter(
-                    pk__in=instance_ids
-                ).values_list("docket_entry__docket_id", flat=True)
+                # Get unique parent_ids for RECAPDocuments
+                parent_ids = list(
+                    set(
+                        RECAPDocument.objects.filter(
+                            pk__in=instance_ids
+                        ).values_list("docket_entry__docket_id", flat=True)
+                    )
```

Comment:
same question about the distinct method here🤔.

Reply:
Oh yeah, I used a `set` because the `distinct` method was not working here due to the nested relation `docket_entry__docket_id`. But you're right is it possible to use `distinct()` here just using `order_by` and setting the field name. Changed.